File: services/reservation/src/main/java/com/example/iter/reservation/dto/response/ReturnTargetResponse.java
package com.example.iter.reservation.dto.response;

import com.example.iter.auth.api.UserSummary;

import java.time.LocalDate;

public record ReturnTargetResponse(
        Long rentalId,
        String equipmentName,
        String thumbnailUrl,
        UserSummary renter,
        LocalDate endDate,
        LocalDate returnDate
) {
}

File: services/reservation/src/main/kotlin/com/example/iter/reservation/dto/response/ReturnTargetResponse.kt
package com.example.iter.reservation.dto.response

import com.example.iter.auth.api.UserSummary
import java.time.LocalDate
import kotlin.jvm.JvmRecord

@JvmRecord
data class ReturnTargetResponse(
    val rentalId: Long,
    val equipmentName: String,
    val thumbnailUrl: String?,
    val renter: UserSummary,
    val endDate: LocalDate,
    val returnDate: LocalDate?
)
